ios/BetterRailWidget/Live Activity/BetterRailLiveActivity.swift

```diff
@@ -23,19 +23,18 @@ struct BetterRailLiveActivity: Widget {
       return DynamicIsland {
             // Expanded UI goes here.  Compose the expanded UI through
             // various regions, like leading/trailing/center/bottom
-          DynamicIslandExpandedRegion(.leading, priority: 90.0) {
+          DynamicIslandExpandedRegion(.leading) {
```

Comment:
Why did you remove the priority?

Reply:
It doesn't have an affect on the view anymore